CITATION.cff

```diff
@@ -0,0 +1,27 @@
+# This CITATION.cff file was generated with cffinit.
+# Visit https://bit.ly/cffinit to generate yours today!
+
+cff-version: 1.2.0
+title: Deploy-to-Grading
+message: >-
+  If you use this software, please cite it using the
+  metadata from this file.
+type: software
+authors:
+  - given-names: Carsten
+    family-names: Gips
+    email: cagix@hsbi.de
+    affiliation: HSBI
+    orcid: 'https://orcid.org/0000-0002-4230-9060'
+  - given-names: André
+    family-names: Kirsch
+repository-code: 'https://github.com/Programmiermethoden/Deploy-to-Grading'
+abstract: >-
+  Deploy to Grading is a toolchain for automated analysis
```

Comment:
```suggestion
  Deploy-to-Grading is a toolchain for automated analysis
```

Ich habe Deploy-to-Grading immer mit Bindestrichen geschrieben. Weiter oben steht es im Titel auch mit Bindestrich. Sollten wir das hier auch so machen?

Reply:
@AKirsch1 macht sinn :) ... dann musst du aber leider nochmal ran :)